src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -109,7 +106,8 @@ class ReviewCollapsibleChapter extends React.Component {
     if (chapterFormConfig.reviewTitle) {
       chapterTitle = chapterFormConfig.reviewTitle;
     }
-    return chapterTitle;
+
+    return chapterTitle || '';
```

Comment:
Are you checking for null/undefined here? This is probably a good area for nullish coalescing `??` @tblackwe reminded me of :)

Reply:
Logical or (`||`)takes care of that:

```
Welcome to Node.js v14.15.0.
Type ".help" for more information.
> undefined || '';
''
> null || '';
''
```

This isn't new logic, I'm just moving it to a reusable place. :)